### PR TITLE
Add MangaTales

### PR DESCRIPTION
**Pull request by MikeZeDev**

### CODE REVIEW

File: web/src/engine/websites/decorators/GManga.ts [hunk 1]
```diff
@@ -0,0 +1,360 @@
...
+
+        return class extends ctor {
+            public async FetchImage(this: MangaScraper, page: Page, priority: Priority, signal: AbortSignal): Promise<Blob> {
+                return FetchImageAjax.call(this, page, priority, signal);
+            }
+        };
+    };
+}
+
+export async function _haqiqa(t: string) : Promise<string>{
```
Comment (Pierre-Monier) [inline web/src/engine/websites/decorators/GManga.ts:293]: Why is this method named `_haqiqa` ? I search on google and it means truth in arabic language, so I'm confused ^^

File: web/src/engine/websites/decorators/GManga.ts [hunk 2]
```diff
@@ -0,0 +1,360 @@
...
+            }),
+            o.push(e);
+        }),
+        o;
+    }
+}
+
+/**
+ *****************
+ * ** END GMANGA ***
```
Comment (Pierre-Monier) [inline web/src/engine/websites/decorators/GManga.ts:358]: I think this comment is a bit useless

File: web/src/engine/websites/decorators/GManga.ts (hunk 1 above)
Comment (Pierre-Monier) [reply web/src/engine/websites/decorators/GManga.ts:293]: Actually, since the main language is Arabic it might make sens

**Comment by Pierre-Monier** [comment]
Looks fine to me (but merge conflict)

File: web/src/engine/websites/decorators/GManga.ts (hunk 1 above)
Comment (MikeZeDev) [reply web/src/engine/websites/decorators/GManga.ts:293]: I guess it was named like that in Gmanga, so it end up like that in the template in Hakuneko :D

**Comment by MikeZeDev** [comment]
> Looks fine to me (but merge conflict)

Yup merge conflict is inevitable at this rate, index.ts is the file were websites classes are imported. We just have to run the index rebuilding script and its settled.

File: web/src/engine/websites/decorators/GManga.ts (hunk 1 above)
Comment (Pierre-Monier) [reply web/src/engine/websites/decorators/GManga.ts:293]: I think renaming this method with something like `_decrypt` is a bit less disturbing ^^

Comment (MikeZeDev) [reply web/src/engine/websites/decorators/GManga.ts:293]: then `decrypt` because _ prefix is meant to be used for unused stuff

**Review by ronny1982 — CHANGES_REQUESTED** [review]

File: web/src/engine/websites/legacy/GManga.ts [hunk 3]
```diff
@@ -1,244 +0,0 @@
...
-// See: https://gist.github.com/ronny1982/0c8d5d4f0bd9c1f1b21dbf9a2ffbfec9
-
-//import { Tags } from '../../Tags';
-import icon from './GManga.webp';
-import { DecoratableMangaScraper } from '../../providers/MangaPlugin';
-
-export default class extends DecoratableMangaScraper {
-
-    public constructor() {
-        super('gmanga', `GManga`, 'https://gmanga.me' /*, Tags.Language.English, Tags ... */);
```
[the remark this answers is not in the document]
Comment (MikeZeDev) [reply web/src/engine/websites/legacy/GManga.ts:11]: Pretty sure its a copycat, cause GMANGA was dmcaed. Ok, ill add it

**Pull request merged**